app/renderer/js/main.ts

```diff
@@ -1210,6 +1210,15 @@ window.addEventListener("load", async () => {
               >${t.__("Do Not Disturb")}</span
             >
           </div>
+          <div id="dnd-dropdown" class="dnd-dropdown hidden">
+            <div class="dnd-dropdown-item" data-minutes="30">30 minutes</div>
+            <div class="dnd-dropdown-item" data-minutes="60">1 hour</div>
+            <div class="dnd-dropdown-item" data-minutes="180">3 hours</div>
+            <div class="dnd-dropdown-item" data-minutes="720">12 hours</div>
+            <div class="dnd-dropdown-item" data-minutes="forever">
+              Until I resume
```

Comment:
These strings need to al be tagged for translation. Look at the adjacent strings for examples of how to do this properly.